### PR TITLE
Update ops handler documentation some more

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #126511
* __->__ #126480

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

**Review by peterbell10 — APPROVED** [review]

File: torch/_inductor/ops_handler.py [hunk 1]
```diff
@@ -447,28 +450,195 @@ def sub(self, x0: T, x1: T) -> T:
...
+    def shifted_chebyshev_polynomial_w(self, x: T, y: T) -> T:
+        ...
+
+    def hermite_polynomial_h(self, x: T, y: T) -> T:
+        ...
+
+    def hermite_polynomial_he(self, x: T, y: T) -> T:
+        ...
+
+    def laguerre_polynomial_l(self, x: T, y: T) -> T:
```
Comment (eellison) [inline torch/_inductor/ops_handler.py:605]: Does this need to be in the ops_handler file ? It is not a builtin op that inductor codegens. Nor it is defined elsewhere.

Comment (eellison) [reply torch/_inductor/ops_handler.py:605]: Nevermind, found where it is referenced.

**Pull request closed without merging**